src/actions/city.actions.js

```diff
@@ -0,0 +1,45 @@
+import { cityConstants, citiesConstants} from '../constants';
+import { citiesService } from '../services';
+
+
+export const cityActions = {
+    deleteCity,
+    getByPlaceId,
+    addCity
+};
+
+
+function deleteCity(city){
+    const deletedCity = citiesService.removeCity(city);
+    return dispatch=>{
+        if(deletedCity){
+            dispatch({type: cityConstants.DELETE, city: deletedCity});
+            return;
+        }
+        dispatch({type: cityConstants.ADD,city:undefined});
+    }
+
+} 
+
+function getByPlaceId(city){
```

Comment:
rename `city` to `placeId` as far as you wait for it.

Reply:
+